labkey/query.py

```diff
@@ -176,8 +174,7 @@ def delete_rows(
 
     return server_context.make_request(
         url,
-        json_dumps(payload, sort_keys=True),
-        headers=_query_headers,
+        json=payload,
```

Comment:
nit: Just a small cleanup item remaining is the unused import for `json_dumps` still remains in these files (e.g. query.py, etc).